src/vueWrapper.ts

```diff
@@ -75,14 +69,10 @@ function createVMProxy<T extends ComponentPublicInstance>(
 }
 
 export class VueWrapper<
-  T extends Omit<
-    ComponentPublicInstance,
-    '$emit' | keyof ComponentCustomProperties
-  > & {
-    $emit: (event: any, ...args: any[]) => void
-  } & ComponentCustomProperties = ComponentPublicInstance
+  T = any,
```

Comment:
Is it normal to not have an `extend` close here?

Reply:
Yes, and I've updated to make it more explicit.